docs/BR.md

```diff
@@ -2104,7 +2104,7 @@ CAs SHALL NOT include additional names unless the CA is aware of a reason for in
 | \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
 | \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
 | \ \ \ \ `validity`             | |
-| \ \ \ \ \ \ \ \ `notBefore`    | __**TBD**__ |
+| \ \ \ \ \ \ \ \ `notBefore`    | A value within 48 hours of the certificate signing operation. |
```

Comment:
I realize this was from https://lists.cabforum.org/pipermail/validation/2021-November/001728.html but I didn't see any explanation/rationale?

Unlike CA signing, there's no need to backdate here, and this seems to violate the existing BRs' 9.6.1

Was the only concern to resolve the precertificate question? Just making sure I can try to address this correctly / offer suggestions here.